neptune/new/internal/init/run.py

```diff
@@ -159,34 +159,34 @@ def init_run(
         >>> import neptune.new as neptune
 
         >>> # minimal invoke
-        ... run = neptune.init()
+        ... run = neptune.init_run()
```

Comment:
NB: These examples are reworked by PR [#1015](https://github.com/neptune-ai/neptune-client/pull/1015) and the change from `init` to `init_run` is reflected there already.

Reply:
Sure. Let me know once it will be merged so I'll be able to easily get rid of it during rebasing.

Reply:
Done